src/license_expression/__init__.py

```diff
@@ -257,6 +257,19 @@ def __init__(self, symbols=tuple(), quiet=True):
 
         # FIXME: this should be instead a super class of all symbols
         self.LicenseSymbol = self.Symbol
+        # LicenseWithExceptionSymbol does not get its internal Expressions mapped durring BooleanAlgebra init
+        # have to set it after the fact
+        tf_nao = {
+            "TRUE": self.TRUE,
+            "FALSE": self.FALSE,
+            "NOT": self.NOT,
+            "AND": self.AND,
+            "OR": self.OR,
+            "Symbol": self.Symbol,
```

Comment:
Should not this be instead? Not sure though.
```suggestion
            "Symbol": self.LicenseWithExceptionSymbol,
```

Reply:
@arbassett-qnx ok, I am going to merge as-is. I you want to take a stab at updating boolean.py that would be awesome!